.gitignore

```diff
@@ -1 +0,0 @@
-/config.php
```

Comment:
do not delete config.php from gitignore

Reply:
Should I commit gitignore files during pull requests? I am kinda new to this, sorry